### PR TITLE
Add experimental warning

### PR DESCRIPTION
**Pull request by josephmure**

### CODE REVIEW

**Review by sofianehaddad — APPROVED** [review]

**Comment by jschueller** [comment]
the warning box is huge, probably needs some css tweaking

File: python/src/FieldFunctionalChaosSobolIndices_doc.i.in [hunk 1]
```diff
@@ -1,6 +1,10 @@
 %feature("docstring") OT::FieldFunctionalChaosSobolIndices
 "Sobol indices from a functional decomposition.
 
+.. warning::
```
Comment (jschueller) [inline python/src/FieldFunctionalChaosSobolIndices_doc.i.in:4]: the warning box are huge, probably needs some css tweaking

File: python/src/FieldToPointFunctionImplementation_doc.i.in [hunk 2]
```diff
@@ -1,6 +1,10 @@
 %define OT_FieldToPointFunction_doc
 "Function mapping a field into a point.
 
+.. warning::
+    This class is experimental and likely to be modified in future releases.
+    To use it, import the ``openturns.experimental`` submodule.
+
```
[the remark this answers is not in the document]
Comment (josephmure) [reply python/src/FieldToPointFunctionImplementation_doc.i.in]: Fixed.

File: python/src/FieldFunctionalChaosSobolIndices_doc.i.in (hunk 1 above)
Comment (josephmure) [reply python/src/FieldFunctionalChaosSobolIndices_doc.i.in:4]: Not sure what to do about this. Does it need to be fixed before the release? It is not too big an issue IMO.

Comment (josephmure) [reply python/src/FieldFunctionalChaosSobolIndices_doc.i.in:4]: Fixed.

**Pull request merged**